src/index.js

```diff
@@ -55,6 +55,12 @@ function decode (data) {
   if (Buffer.isBuffer(data)) {
     data = data.toString()
   }
+  const prefix = data[0]
+
+  // Make all encodings case-insensitive expect the ones that include upper and lower chars in the alphabet
+  if (!['z', 'Z', 'm', 'M', 'u', 'U', '\x00'].includes(prefix)) {
```

Comment:
Having this be a blacklist makes me uneasy. I would have written it as an explicit whitelist instead ( i.e. LC only encodings that are explicitly specified )

Reply:
is this simply a blacklist vs whitelist thing ?

Reply:
Typo: expect -> except.

I'd also use a whitelist, but I don't have a strong opinion about it.